docs/articles/documentation/recipes/best-practices/create-faster-tests.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: docs
+title: Create Faster Tests
+permalink: /documentation/recipes/best-practices/create-faster-tests.html
+redirect_from:
+  - /documentation/recipes/create-faster-tests.html
+---
+# Speed Up Your Tests
+
+This section describes ways to decrease the execution time of TestCafe tests.
+
```

Comment:
What about adding the list of all described ways to provide customer a brief overview:

```js
* [run tests concurrently](http://link1)
* [run tests in headless browsers](http://link2)

```

Reply:
I thought the article would be too small for that kind of thing. But it has grown quite significantly, so it's a good suggestion